Source/Logic/SpeechService/SpeechService.swift

```diff
@@ -116,13 +134,11 @@ class SpeechService: NSObject, SpeechServiceProtocol, SFSpeechRecognizerDelegate
                 self.stopRecording()
                 self.dongSound.play()
                 self.delegate?.speechRecognitionFinished(transcription: result!.bestTranscription.formattedString)
```

Comment:
Which component is doing the silence detection during the speech rec?  Is Snowboy doing that or the Apple Speech API does that for you automatically?  I believe when I did it, Snowboy handled it.

Reply:
We're doing it via the Speech API. I tried doing it via Snowboy as well, but didn't see any significant changes.